swagger.yaml

```diff
@@ -3249,6 +3290,54 @@ definitions:
       worker:
         $ref: '#/definitions/host-requirements-role'
 
+  preflight-requirements:
+    type: object
+    properties:
+      operators:
+        description: Preflight operators requirements
+        type: array
+        items:
+          $ref: '#/definitions/operator-requirements'
+      ocp:
```

Comment:
Can we use a more indicative name? Maybe something that includes the word `hardware`?

Reply:
Good idea.